Update swim-lessons.md

ALFalls opened the pull request:
Edits @sspicer1 @SharonManyara

ALFalls committed:
Update swim-lessons.md

jhutchings1 commented:
👋🏻  Hey @ALFalls Just an FYI that the staging site only keeps the most recent branch available for testing. If you'll be making many updates in a day, I recommend putting them all on a single branch and testing them together.

jhutchings1 commented on _amenities/pools/swim-lessons.md:
```diff
@@ -9,30 +9,32 @@ show-title: true
+  
+ Klahanie pools are open, and swim lessons are available. For the safety of our residents and staff, we follow CDC guidelines for the prevention of COVID-19. 
+ 
+<strong>Quick Links</strong> 
```
What are you trying to accomplish with this change? This moves the Quick links header out of the floating card on the right side of the page and puts it into the yellow alert banner.

ALFalls commented:
> 👋🏻 Hey @ALFalls Just an FYI that the staging site only keeps the most recent branch available for testing. If you'll be making many updates in a day, I recommend putting them all on a single branch and testing them together.

@jhutchings1 Sure. Can you please send instructions for doing so or point me to a tutorial that gives steps? Thx.

SharonManyara committed:
Update swim-lessons.md

@jhutchings1 I think i corrected the issue. Please let me know if this is ready to be published. 
@sspicer1

SharonManyara reviewed (changes requested):
Attempting to correct the "Note" code problem.

jhutchings1 commented:
If you look at the diff here, you'll see that there are still changes to the menu on line 16. You should put it back like it was to avoid changing that menu. https://github.com/Klahanie/klahanie/pull/97/files

Also, @SharonManyara , it looks like you selected `request changes`. Please be aware that that choice blocks the pull request until you change your vote to `approve`. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews

SharonManyara committed:
Update swim-lessons.md

@jhutchings1 I corrected the changes that Autumn accidentally made.
